common/djangoapps/student/cookies.py

```diff
@@ -5,6 +5,8 @@
 import time
 import json
 
+from email_marketing.signals import add_email_marketing_cookies
```

Comment:
Perhaps `add_email_marketing_cookies` should move to a non-signals related file? `email_marketing.cookies`?